assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -27,7 +27,7 @@ const Background = styled.div`
     ".    prev   page       . next     ." ${ PAGE_HEIGHT }px
     ".    .      menu       . .        ." 48px
     ".    .      carrousel  . .        ." 1fr
-    / 1fr ${ LEFT_NAV_WIDTH }px ${ PAGE_WIDTH }px ${ PAGE_NAV_PADDING }px 1fr ${ CENTRAL_RIGHT_PADDING }px;
+    / 1fr ${ LEFT_NAV_WIDTH }px ${ PAGE_WIDTH }px ${ PAGE_NAV_PADDING }px 1fr 1fr;
```

Comment:
Since we removed "add page", what's that other column now between "page" and "next"? Also "1fr" for carousel seems wrong. You could probably bump it down to later, so let me know. But overall this layout seems not quite right.

Reply:
I killed central right padding so it's hard for me to revert this, and central right padding is the only change here. Would be good to merge, and then I'll leave it alone?